### PR TITLE
[SDK] Remove line number from flow as func output

### PR DESCRIPTION
**Pull request by D-W-**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: examples/tutorials/get-started/quickstart-azure.ipynb [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "- An Azure account with an active subscription - [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F)\n",
     "- An Azure ML workspace - [Configure workspace](../../configuration.ipynb)\n",
     "- A python environment\n",
-    "- Installed PromptFlow SDK\n",
+    "- Installed prompt-flow SDK\n",
```
Comment (wangchao1230) [inline examples/tutorials/get-started/quickstart-azure.ipynb:13]: prompt-flow -> prompt flow

File: src/promptflow/promptflow/_sdk/operations/_test_submitter.py [hunk 2]
```diff
@@ -258,7 +258,7 @@ def exec_with_inputs(self, inputs):
         )
         flow_executor.enable_streaming_for_llm_flow(lambda: self.flow_context.streaming)
         line_result = flow_executor.exec_line(inputs, index=0, allow_generator_output=self.flow_context.streaming)
-        return line_result
+        return line_result.output
```
Comment (wangchao1230) [inline src/promptflow/promptflow/_sdk/operations/_test_submitter.py:261]: shall we add a test assert this?

File: examples/tutorials/get-started/quickstart-azure.ipynb [hunk 3]
```diff
@@ -426,7 +426,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.17"
+   "version": "3.8.13"
```
Comment (zhengfeiwang) [inline examples/tutorials/get-started/quickstart-azure.ipynb:429]: @crazygao I remember you mentioned we need to keep this unchanged? but I forget the reason...

Comment (crazygao) [reply examples/tutorials/get-started/quickstart-azure.ipynb:429]: No special reasons, we don't need to change this property.

File: src/promptflow/promptflow/_sdk/operations/_test_submitter.py (hunk 2 above)
Comment (D-W-) [reply src/promptflow/promptflow/_sdk/operations/_test_submitter.py]: added in test

File: examples/tutorials/get-started/quickstart-azure.ipynb (hunk 3 above)
Comment (D-W-) [reply examples/tutorials/get-started/quickstart-azure.ipynb]: good catch, reverted

**Review by wangchao1230 — APPROVED** [review]

**Review by crazygao — APPROVED** [review]

**Pull request merged**